Lib/fontmake/__main__.py

```diff
@@ -156,6 +156,11 @@ def main(args=None):
         'E.g.: -i "Noto Sans Bold"; or -i ".* UI Condensed". '
         "(for Glyphs or MutatorMath sources only). ",
     )
+    outputGroup.add_argument(
+        "--use-mutatormath",
+        action="store_true",
+        help="Use MutatorMath to generate instances.",
```

Comment:
maybe mention here in the help section for `--use-mutatormath` why one would like to use that, e.g. if output relies on extrapolation or anisotropic interpolation (not supported by varLib), or why _not_ to use it (doesn't support rules or sparse layers).